lib/setup.py

```diff
@@ -38,10 +51,10 @@
                 on Sedna project",
     packages=find_packages(exclude=["tests", "*.tests",
                                     "*.tests.*", "tests.*"]),
-    author="",
-    author_email="",
-    maintainer="",
-    maintainer_email="",
+    author=maintainer,
+    author_email="pujie2@huawei.com",
+    maintainer=maintainer,
+    maintainer_email="liulinghui@huawei.com",
```

Comment:
put your email instead of mine, or just remove mine.

Reply:
check